### DISCUSSION TITLE
Vector Store Pinecone: Pinecone Retriever vs Pinecone Vector Store

### DISCUSSION THREAD

**Discussion by koayst** [FlowiseAI/Flowise#2188] [Q&A]
For Vector Stores, specifically Pinecone, the output can either be "Pinecone Retriever" or "Pinecone Vector Store".  What is the difference?  Can you give me examples when to use which ?

Thanks.

**Comment by edvinPL** [comment]
having the same doubt :)

**Comment by gnotus** [comment]
Pinecone Vector Store: Focuses on storage, management, and maintenance of vectors and their associated metadata. Use it when you need to store, update, or manage vector data.
Pinecone Retriever: Focuses on the retrieval and search functionalities, abstracting the querying process. Use it when you need to perform semantic searches or integrate with machine learning models for context-aware retrieval.
